.eslintrc.js

```diff
@@ -3,20 +3,24 @@ module.exports = {
     browser: false,
     commonjs: true,
     es2020: true,
-    "jest/globals": true,
-  },
-  extends: [
-    "eslint:recommended",
-    "plugin:node/recommended",
-    "plugin:package-json/recommended",
-  ],
-  plugins: ["package-json", "jest"],
-  globals: {
-    Atomics: "readonly",
-    SharedArrayBuffer: "readonly",
   },
+  extends: ["eslint:recommended", "plugin:node/recommended"],
   parserOptions: {
-    ecmaVersion: 2019,
+    ecmaVersion: 2020,
   },
-  rules: {},
+  overrides: [
+    {
+      files: ["package.json", "**/package.json"],
```

Comment:
I found this didn't appear to be working before/after this, but I left it anyway. Think it's on an older ESLint plug-in setup that isn't working in v7